### PR TITLE
Added Blackjack

### LINKED ISSUE #549: Blackjack
# Aim

- One will get proper understanding of OOPS (Object-Oriented Programming System) by implementing this type of script as it contains classes and functions from scratch.

# Details

- It will contain following piece of code:
1. import modules and define variables
2. make classes
3. make functions
4. game endings
5. game play

The output will look like this:

bj1
bj2

### PR DESCRIPTION
**Pull request by mehabhalodiya**
# Description

While building the game, one will explore a few handy Python concepts, such as object-oriented programming using classes and how to manage a game loop.

I have used Python's `random` library, which has various functions used in generating randomness. In particular, we employ `shuffle`, which takes any list and returns it in random order, to shuffle our deck of cards.

Initially, we will begin by defining the classes that will be used in order to separate out different aspects of the game of blackjack. We will model three of the components of the game:

- `Card`: A basic playing card. The card belongs to a suit (hearts ♥, diamonds ♦, spades ♠, or clubs ♣) and is worth a certain value.
- `Deck`: A collection of cards. The deck shrinks as cards are drawn and contains 52 unique cards.
- `Hand`: Each player's assigned cards. A hand is what defines each player's score and thus who wins.
- `Chips`: To keep track.

Further, create all the list of functions which perform various actions and finally, the game loop!

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots that prove my script is effective or that my feature works.

### CODE REVIEW

File: Blackjack/blackjack.py [hunk 1]
```diff
@@ -0,0 +1,220 @@
+# IMPORT MODULES AND DEFINE VARIABLES
+
+import random
+
+suits = ('Hearts', 'Diamonds', 'Spades', 'Clubs')
```
Comment (kaustubhgupta) [inline Blackjack/blackjack.py:5]: make the suits and ranks as class variables of Card class and values in Hand class

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: Blackjack/blackjack.py (hunk 1 above)
Comment (mehabhalodiya) [reply Blackjack/blackjack.py]: Done!

**Comment by kaustubhgupta** [comment]
@mehabhalodiya I guess I made an error in the suggestion:
The suits and ranks were part of Deck class and not Card class. Also, they need to be accessed via class name:
```python
class Deck:  # creates a deck of cards

    suits = ('Hearts', 'Diamonds', 'Spades', 'Clubs')
    ranks = ('Two', 'Three', 'Four', 'Five', 'Six', 'Seven', 'Eight', 'Nine', 'Ten', 'Jack', 'Queen', 'King', 'Ace')

    def __init__(self):
        self.deck = []  # haven't created a deck yet
        for suit in Deck.suits:
            for rank in Deck.ranks:
                self.deck.append(Card(suit, rank))
```

Implement the same in Hand class.

**Comment by mehabhalodiya** [comment]
So i have to remove `suits` and `ranks` from `Card` class and have to add in `Deck` class?
Further, `suits` and `ranks` should be added in `Hand` class too. Right?

**Comment by kaustubhgupta** [comment]
> So i have to remove `suits` and `ranks` from `Card` class and have to add in `Deck` class?

Yes

> Further, `suits` and `ranks` should be added in `Hand` class too. Right?

Nope, only values in Hand class.

Make sure to update the access method:
```python
 def __init__(self):
        self.deck = []  # haven't created a deck yet
        for suit in Deck.suits:
            for rank in Deck.ranks:
```

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
@mehabhalodiya did you run the script after updating?

**Comment by mehabhalodiya** [comment]
Yes, I have checked it and it is now showing error messages as below:

bj

**Comment by kaustubhgupta** [comment]
> Yes, I have checked it and it is now showing error messages as below:
> 
> bj

Exactly, this error will pop up because as you made the "values" class variable, it will be accessed as Class_name.values. Mentioned this in https://github.com/avinashkranjan/Amazing-Python-Scripts/pull/591#issuecomment-800137709 but you updated it for one class only

**Comment by mehabhalodiya** [comment]
> > Yes, I have checked it and it is now showing error messages as below:
> > bj
> 
> Exactly, this error will pop up because as you made the "values" class variable, it will be accessed as Class_name.values. Mentioned this in [#591 (comment)](https://github.com/avinashkranjan/Amazing-Python-Scripts/pull/591#issuecomment-800137709) but you updated it for one class only

Sorry for this! I forgot!
Now I have modified.

**Review by kaustubhgupta — APPROVED** [review]

**Comment by mehabhalodiya** [comment]
@antrikshmisri @santushtisharma10 Please review this!

**Review by antrikshmisri — APPROVED** [review]
LGTM 💯

**Review by avinashkranjan — APPROVED** [review]
LGTM..Great Work

**Pull request merged**